valid-anagram/JoyJaewon.py

```diff
@@ -0,0 +1,43 @@
+'''
+Brainstorm
+- brute force: double for-loop O(N^2)
+- efficient: use hash map O(N)
+
+plan
+1. check the string lengths. Return false if different
+2. Initialize the dictionary
+3. Check the dictionary
+    3-1. If all counts are zero, strings are anagrams. Return True
+    3-2. If not, return False
+'''
+def isAnagram(s, t):
+    if len(s) != len(t):
+        return False
+    
+    count = {}
+    for char in s:
+        if char in count:
+            count[char] += 1
+        else:
+            count[char] = 1
```

Comment:
참고: 요렇게 한 줄로 짜실 수도 있습니다.

```suggestion
        count[char] = count.get(char, 0) + 1
```